samples/ManagementSite/Alloy.ManagementSite.csproj

```diff
@@ -43,6 +41,9 @@
     <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="6.0.0" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="6.0.0" />
     <PackageReference Include="Microsoft.Extensions.Http.Polly" Version="6.0.0" />
+    <PackageReference Include="Optimizely.Headless.Form.ContentGraph" Version="0.1.1-inte-380" />
+    <PackageReference Include="Optimizely.Headless.Form.Core" Version="0.1.1-inte-380" />
```

Comment:
No need reference Core package directly since we referencing Service package.
Should use variable version for easier control.